packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx

```diff
@@ -0,0 +1,64 @@
+import CheckOutlinedIcon from '@mui/icons-material/CheckOutlined';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+import {
+  Box,
+  Button,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  TextField,
+  Typography,
+} from '@mui/material';
+import React, { useCallback, useState } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```

Comment:
Food for thought: do we really need the `CopyToClipboard` library? https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API? Whatever you choose it's fine